#!/bin/bash

PATCH="mimeinfo-file-manager.patch"
PACKAGE="x11-misc/shared-mime-info"
for i in $PACKAGE/*.ebuild; do
	cat > ${PATCH} << EOF
--- a/${i}
+++ b/${i}
@@ -45,5 +45,10 @@
 src_install() {
 	default
 
+	cat >> \${ED}/usr/share/applications/defaults.list << EOF
+[Default Applications]
+inode/directory=file-manager.desktop
+EOF
+
 	# in prefix, install an env.d entry such that prefix patch is used/added
 	if use prefix; then
EOF
	git apply ${PATCH} || exit 1
	rm ${PATCH}
done
